src/cli/commands/bitswap.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+module.exports = {
+  command: 'bitswap',
+
+  description: 'A set of commands to manipulate the bitswap agent.',
+
+  builder (yargs) {
+    return yargs
+      .commandDir('bitswap')
```

Comment:
Yes this this is the recommended (and as far as I can tell only way) to do this.